### ISSUE TITLE
rclone config of Seafile library fails (Error: two-factor authentication is not enabled on this account)

### ISSUE THREAD

**Issue by lnnrtwttkhn** [rclone/rclone#5660]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/seafile-fatal-error-two-factor-authentication-is-not-enabled-on-this-account/26771

#### What is the problem you are having with rclone?

Starting in rclone version v1.56.0, `rclone config` of a Seafile library results in an error apparently related to two-factor authentication:

```bash
Error: two-factor authentication is not enabled on this account
Usage:
  rclone config create `name` `type` [`key` `value`]* [flags]
Flags:
      --all               Ask the full set of config questions.
      --continue          Continue the configuration process with an answer.
  -h, --help              help for create
      --no-obscure        Force any passwords not to be obscured.
      --non-interactive   Don't interact with user and return questions.
      --obscure           Force any passwords to be obscured.
      --result string     Result - use with --continue.
      --state string      State - use with --continue.
Use "rclone [command] --help" for more information about a command.
Use "rclone help flags" for to see the global flags.
Use "rclone help backends" for a list of supported services.
2021/09/30 12:03:17 Fatal error: two-factor authentication is not enabled on this account
```

This is the `rclone config` command I use:

```bash
rclone config create $CONFIG_NAME seafile url $SEAFILE_URL user $MY_EMAIL_ADDRESS library $LIBRARY_NAME pass $PASSWORD
```

I've tried to add `2fa false` and `seafile-2fa` false but this did not change anything.

#### What is your rclone version (output from `rclone version`)

I tested several versions starting from v1.56.1.
The error starts to appear in v1.56.0.
v1.55.1 and earlier versions work as expected.

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

I run rclone inside a Docker container. The container is using Debian GNU/Linux 11:

```bash
 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

<details>
  <summary>Click here for the Dockefile</summary>

```Dockerfile
FROM python:3-slim

ARG DOCKER_TAG

ENV DEBIAN_FRONTEND=noninteractive

RUN apt-get -y update
RUN apt-get -y install --no-install-recommends locales curl gnupg
RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
RUN locale-gen

# Add neurodebian sources for git-annex-standalone
RUN curl <URL_REMOVED>  > /etc/apt/sources.list.d/neurodebian.sources.list
RUN apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 0xA5D32F012649A5A9
RUN apt-get -y update

# Rclone is a useful add-on
RUN curl -O <URL_REMOVED> && dpkg -i rclone*deb && rm -f rclone*deb

# install packages
RUN apt-get -y install --no-install-recommends git git-annex-standalone git-annex-remote-rclone make

RUN pip3 install "datalad${DOCKER_TAG+==$DOCKER_TAG}"

# clean up
RUN apt-get clean

RUN git config --global user.name "Docker Datalad"
RUN git config --global user.email "docker-datalad@example.com"

ENTRYPOINT ["datalad"]
```

</details>

#### Which cloud storage system are you using? (e.g. Google Drive)

Seafile

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

```bash
rclone config create $CONFIG_NAME seafile url $SEAFILE_URL user $MY_EMAIL_ADDRESS library $LIBRARY_NAME pass $PASSWORD
```

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

```
$ rclone config create $CONFIG_NAME seafile url $SEAFILE_URL user $MY_EMAIL_ADDRESS library $LIBRARY_NAME pass $PASSWORD
2021/09/30 12:03:17 NOTICE: Config file "/root/.config/rclone/rclone.conf" not found - using defaults
Error: two-factor authentication is not enabled on this account
Usage:
  rclone config create `name` `type` [`key` `value`]* [flags]
Flags:
      --all               Ask the full set of config questions.
      --continue          Continue the configuration process with an answer.
  -h, --help              help for create
      --no-obscure        Force any passwords not to be obscured.
      --non-interactive   Don't interact with user and return questions.
      --obscure           Force any passwords to be obscured.
      --result string     Result - use with --continue.
      --state string      State - use with --continue.
Use "rclone [command] --help" for more information about a command.
Use "rclone help flags" for to see the global flags.
Use "rclone help backends" for a list of supported services.
2021/09/30 12:03:17 Fatal error: two-factor authentication is not enabled on this account
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Any ideas what is going on here @creativeprojects ?

**Comment by creativeprojects** [comment]
I can see in the code that we do return an error in the 2fa config callback if the option is set to false, so I'm thinking this function maybe wasn't called before although the caller wouldn't know if it's needed of not. Weird.

Anyway, I'll post a fix soon-ish.

**Comment by creativeprojects** [comment]
This is fixed by #5665 I don't know why the merge didn't close it automatically 👍🏻
